waspc/src/Wasp/Generator.hs

```diff
@@ -35,7 +35,7 @@ writeWebAppCode wasp dstDir compileOptions = do
   writeFileDrafts dstDir (generateWebApp wasp compileOptions)
   ServerGenerator.preCleanup wasp dstDir compileOptions
   writeFileDrafts dstDir (genServer wasp compileOptions)
-  DbGenerator.preCleanup wasp dstDir compileOptions
+  _ <- DbGenerator.prepare wasp dstDir compileOptions
```

Comment:
Could be a better name here... no longer just "cleanup" to me since we also copy down. Setup perhaps? Ok leaving if no strong options but wanted to call out for help if there are any better ideas :D Thanks